### PR TITLE
planner: support creating universal bindings

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve?

Issue Number: ref #48875

Problem Summary: planner: support creating universal bindings

### What changed and how does it work?

Implement the syntax in TiDB:
```
CREATE [SESSION | GLOBAL] UNIVERSAL BINDING ...
```

1. Hide schema name for universal bindings;
2. Add a new column named `type` to indicate whether this binding is a universal binding, '' for normal and 'u' for universal bindings;
image

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by elsa0520 — APPROVED** [review]
LGTM

File: pkg/session/bootstrap.go [hunk 1]
```diff
@@ -2933,6 +2939,13 @@ func upgradeToVer180(s sessiontypes.Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.tidb_background_subtask ADD INDEX idx_exec_id(exec_id)", dbterror.ErrDupKeyName)
 }
 
+func upgradeToVer181(s sessiontypes.Session, ver int64) {
+	if ver >= version181 {
+		return
+	}
+	doReentrantDDL(s, "ALTER TABLE mysql.bind_info ADD COLUMN `type` VARCHAR(64) NOT NULL DEFAULT ''", infoschema.ErrColumnExists)
```
Comment (AilinKid) [inline pkg/session/bootstrap.go:2946]: why not use a boolean to represent it

**Review by AilinKid — APPROVED** [review]
Rest LGTM

File: pkg/session/bootstrap.go (hunk 1 above)
Comment (qw4990) [reply pkg/session/bootstrap.go:2946]: I'm afraid we'll introduce more types in the future, so use `type varchar` here.

**Review by tangenta — APPROVED** [review]

**Pull request merged**